force-app/main/notification/classes/HOT_UserNotificationService.cls

```diff
@@ -242,8 +242,8 @@ public without sharing class HOT_UserNotificationService {
         }
     }
 
-    public static void newWantedSRNotification(CustomNotificationType notificationType, Set<String> recipients) {
-        String title = 'Du er ønsket til et oppdrag';
+    public static void newWantedSRNotification(CustomNotificationType notificationType, Set<String> recipients, String appointmentNumber) {
+        String title = 'Du er ønsket til et oppdrag '+appointmentNumber;
```

Comment:
Burde det ikke vært "," eller "." etter "oppdrag". Samme kommentar for HOT_ReminderSMSService også.